### ISSUE TITLE
New bug with renderd text

### ISSUE THREAD

**Issue by TaunT** [ogkalu2/comic-translate#177]
text selection or cursor placement in an already rendered block works glitchily, constantly jumping to the end of the text

**Comment by ogkalu2** [comment]
Does this happen everytime ? or randomly ?

**Comment by TaunT** [comment]
everytime

**Comment by TaunT** [comment]
@ogkalu2 It seems to me that all these bugs with bold text and highlighting are related to the new feature - rotation. Sometimes you click on a block to select, but rotation is turned on, and after that the block becomes glitchy.
In general, rotation is very annoying, sometimes you turn it by accident, and you can't align it back like that. I would like to be able to turn off this feature, because it is rarely needed, and it causes a lot of problems.

**Comment by TaunT** [comment]
![image](https://github.com/user-attachments/assets/317f4c4b-63e7-43e0-8ae2-f64bcda0da6e)
Are you talking about these buttons? They don't seem to work on my version. I'll check on the latest version.

**Comment by ogkalu2** [comment]
I think i finally figured out and fixed what was causing this. At least, i'm no longer having the issue

**Comment by ogkalu2** [comment]
> ![image](https://github.com/user-attachments/assets/317f4c4b-63e7-43e0-8ae2-f64bcda0da6e) Are you talking about these buttons? They don't seem to work on my version. I'll check on the latest version.

Yeah you should be able to under a rotation by clicking on it. Try clicking multiple times and see if it works at all. That may be the issue
